templates/docs/examples/layouts/documentation-full-width.html

```diff
@@ -0,0 +1,422 @@
+                <li class="p-list__item"><a href="#docs/k8s-cloud">Using Kubernetes with Juju&nbsp;›</a></li>
+                <li class="p-list__item"><a href="#docs/k8s-charms-tutorial">Understanding Kubernetes charms&nbsp;›</a></li>
+              </ul>
+            </div>
+            <div class="col-4">
+              <ul class="p-list">
+                <li class="p-list__item"><a href="#docs/migrating-models">Migrating models&nbsp;›</a></li>
+                <li class="p-list__item"><a href="#docs/bundle-reference">Bundle reference&nbsp;›</a></li>
+                <li class="p-list__item"><a href="#docs/microk8s-cloud">Using Juju with MicroK8s&nbsp;›</a></li>
+                <li class="p-list__item"><a href="#docs/removing-things">Removing things&nbsp;›</a></li>
+                <li class="p-list__item"><a href="#docs/controller-logins">Controller logins&nbsp;›</a></li>
+              </ul>
+            </div>
+          </div>
+
+          <p>For three columns split use three <code>col-3</code> columns.</p>
+
+          <div class="row">
+            <div class="col-3">
+              <ul class="p-list--divided">
+                <li class="p-list__item"><a href="#docs/settings/animation-settings">Animation</a></li>
+                <li class="p-list__item"><a href="#docs/settings/assets-settings">Assets</a></li>
+                <li class="p-list__item"><a href="#docs/settings/breakpoint-settings">Breakpoint</a></li>
+              </ul>
+            </div>
+            <div class="col-3">
+              <ul class="p-list--divided">
+                <li class="p-list__item"><a href="#docs/settings/color-settings">Color</a></li>
+                <li class="p-list__item"><a href="#docs/settings/font-settings">Font</a></li>
+                <li class="p-list__item"><a href="#docs/settings/layout-settings">Layout</a></li>
+              </ul>
+            </div>
+            <div class="col-3">
+              <ul class="p-list--divided">
+                <li class="p-list__item"><a href="#docs/settings/placeholder-settings">Placeholder</a></li>
+                <li class="p-list__item"><a href="#docs/settings/spacing-settings">Spacing</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </div>
+  </main>
+
+  <div class="l-docs__footer">
+    <footer class="p-strip--dark l-docs__container">
+      <div class="l-docs__sidebar">
+        <p style="padding-left: 1.5rem">© 2020 Canonical Ltd.</p>
+      </div>
+      <div class="l-docs__main">
+        <nav class="row" aria-label="Footer">
+          <div class="has-cookie">
+            <p><a class="is-dark" href="#">Ubuntu</a> and <a class="is-dark" href="#">Canonical</a> are registered trademarks of Canonical Ltd.</p>
+            <ul class="p-inline-list--middot">
+              <li class="p-inline-list__item">
+                <a class="is-dark" href="#"><small>Legal information</small></a>
```

Comment:
why do we leave links that don't lead anywhere? My expectation would be to have read addresses to take users to when they click on these links.